app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
 end
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the User model has a method that returns 3 most recent posts as stated in the project requirement.